buckaroo/SourceExplorer.fs

```diff
@@ -30,6 +30,7 @@ module SourceExplorer =
         yield!
           xs
           |> List.distinct
+          |> List.sortDescending
```

Comment:
In case of Any( SemVer=1, SemVer=2, ....) we want to make sure we try the latest version first